static/app/app/controller.js

```diff
@@ -59,6 +60,34 @@ var Controller = Marionette.Object.extend({
             spots.fetch().done(callback);
         }
 
+    },
+    join: function(moveId) {
+        var moves = channel.request('entities:moves');
+        
+        $.when(moves.fetch()).done(function(){
+            var currentMove = channel.request("entities:move");
+            var moveToJoin = channel.request('entities:moves')
+                                    .find({id: parseInt(moveId)});
+            
+            currentMove.set('time', moveToJoin.get('time').format());
+            currentMove.set('spot', moveToJoin.get('spot'));
+            
+            if (currentMove.get('user')) {
+                currentMove.save({}, {
+                    dataType: 'text',
+                    success: function(model, resp) {
+                        channel.trigger('list');
+                        console.log(model);
+                    },
+                    error: function(model, resp) {
+                        console.log(resp);
+                    }
```

Comment:
Make sure to take out the `console.log`s as we don't want those in production. Also as we're not doing anything with the error callback at the moment you can just leave it out. Adding in better error handling throughout Lunchmove can be done as a separate PR.